### ISSUE TITLE
Liquid-Rails doesn't pull templates from resolvers

### ISSUE THREAD

**Issue by Stickiler** [chamnap/liquid-rails#21]
With an app we're currently building, we use a Panoramic resolver to pull our templates from the database instead of the filesystem, as we allow templates to be changed on the fly. 

Our main problem is that liquid-rails doesn't pull templates using the resolver, for any templates included inside other templates. The main template called from the rails controller itself pulls from the database fine, but the partials that are included inside that template continue to pull from the filesystem instead.

Any thoughts on ways to get this working?

**Comment by sonnypgs** [comment]
Having the same kind of problem at the moment. Have you found working a solution? @Stickiler

**Comment by Stickiler** [comment]
@sonnypgs Liquid-Rails has it's own resolver definition in it's code that it uses instead of using the Rails resolvers. The solution I ended up needing was forking the liquid-rails repo, and modifying some of the code to properly use the rails resolvers instead of hard-coding a filesystem resolver for the templates. I started with the code from #9 , then added the new Liquid::Rails::ResolverSystem file as the default resolver in the liquid-rails code.
